amethyst_ui/src/transform.rs

```diff
@@ -115,6 +132,27 @@ impl UiTransform {
         self.opaque = false;
         self
     }
+
+    /// Adds stretching to this ui element so it can fill its parent.
+    pub fn with_stretching(mut self, stretch: Stretch) -> Self {
```

Comment:
`with_stretch` ?

Might wanna add `with_anchor` also

Reply:
I felt `with_anchor` was unnecessary because the field is public and I ask for the anchor in the constructor.  I'll fix the naming though.